app/frontend/components/pages/sendAda/confirmTransactionDialog.tsx

```diff
@@ -260,6 +268,7 @@ export default connect(
     rawTransactionOpen: state.rawTransactionOpen,
     stakePool: state.shelleyDelegation.selectedPool,
     txConfirmType: state.txConfirmType,
+    poolRecommendation: getSourceAccountInfo(state).poolRecommendation,
```

Comment:
We should probably remove the connect altogether and not use both connect and useSelector. But we can also refactor this project-wise later.

Reply:
That's forgotten for sure, I'll remove it